clang/include/clang/DPCT/DPCTOptions.inc

```diff
@@ -181,7 +181,7 @@ DPCT_NON_ENUM_OPTION(DPCT_OPT_TYPE(static llvm::cl::opt<bool, true>), CheckUnico
            llvm::cl::cat(DPCTCat), llvm::cl::location(CheckUnicodeSecurityFlag))
 
 DPCT_NON_ENUM_OPTION(DPCT_OPT_TYPE(static llvm::cl::opt<bool, true>), EnablepProfiling, "enable-profiling",
-           llvm::cl::desc("Enable SYCL queue profiling in helper functions. Default: off.\n"),
+           llvm::cl::desc("Enable SYCL queue profiling in helper functions on demand. \n"),
```

Comment:
default: auto  (auto: mean off: if tool conclude profiling is not required, on: if tool conclude profiling is required).